app/models/post.rb

```diff
@@ -2,4 +2,12 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
+
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
 end
```

Comment:
- [ ] Please add a callback to be called after a record is saved, then make the `update_user_posts_counter method private.
e.g
```
  def recent_comments(limit = 5)
    comments.order(created_at: :desc).limit(limit)
  end

  after_save :update_user_posts_counter

  private

  def update_user_posts_counter
    author.update(posts_counter: author.posts.count)
  end
 
```